apps/webapp/src/app/ical/ical.tsx

```diff
@@ -71,9 +71,9 @@ export const Ical = (props: {
 
 function dateArray(
   date: string,
-  timeFrom: Date | string | number | (number | string)[]
+  time: Date | string | number | (number | string)[]
```

Comment:
timeFrom było niefajne? bo jak teraz widzę time to nie wiem czy to from, to, czas trwania. zgaduję, że time of start ale to już moja domysły na temat aplikacji?

Reply:
ale to jest jednocześnie i timeFrom i timeTo, bo ta metoda jest reużyta :thinking:

Reply:
ok :D